testing/integration-tests/build.rs

```diff
@@ -0,0 +1,9 @@
+use cfg_aliases::cfg_aliases;
+
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases! {
+        lightclient: { any(feature = "unstable-light-client", feature = "unstable-light-client-long-running") },
```

Comment:
I would prefer `light-client` here if that's possible

Reply:
Ahh this doesn't work with `cfg_aliases`, it hits recursion limit for whatever reason.

From https://github.com/katharostech/cfg_aliases?tab=readme-ov-file#syntax-and-error-messages:
> The aliase names are restricted to the same rules as rust identifiers which, for one, means that they cannot have dashes ( - ) in them